twofactor_privacyidea/js/push.js

```diff
@@ -0,0 +1,14 @@
+window.onload=function(){
+    setInterval(function() {
+        $.ajax({
+            url:location.href,
+            type:'post',
+            data: $('#piLoginForm').serialize(),
+            success:window.onload=function(result){
+                if(result.search("<span class=\"warning\">") === -1){
+                    location.reload()
+                }
+            }
+        })
+    }, 2000);
```

Comment:
Can we do it with a one second interval?

Reply:
Yes we could do. We must performance enhancements anyway, see privacyidea/privacyidea#1701. Changed.